cmd/kubeadm/app/master/clusterroles.go

```diff
@@ -0,0 +1,47 @@
+package master
+
+import (
+	"fmt"
+
+	api "k8s.io/kubernetes/pkg/api/v1"
+	rbac "k8s.io/kubernetes/pkg/apis/rbac/v1alpha1"
+	"k8s.io/kubernetes/pkg/client/clientset_generated/clientset"
+)
+
+// CreateBootstrapRBACClusterRole creates the necessary ClusterRole for bootstrapping
+func CreateBootstrapRBACClusterRole(clientset *clientset.Clientset) error {
+	clusterRole := rbac.ClusterRole{
+		// a role to use for setting up a proxy
+		ObjectMeta: api.ObjectMeta{Name: "system:kubelet-bootstrap"},
+		Rules: []rbac.PolicyRule{
+			rbac.NewRule("get").Groups("").Resources("nodes").RuleOrDie(),
+			rbac.NewRule("create", "watch").Groups("certificates.k8s.io").Resources("certificatesigningrequests").RuleOrDie(),
```

Comment:
@deads2k what would you think about making this an actual bootstrap role?

@andrewrynhard I'm unsure about using the `system:` namespace for things outside the core bootstrap roles.